docs/docsite/rst/common/settings-menu.rst

```diff
@@ -8,5 +8,6 @@
 To enter the Settings window for AWX, click **Settings** from the left navigation bar. This page allows you to modify your AWX configuration, such as settings associated with authentication, jobs, system, and user interface.
 
 .. image:: ../common/images/ug-settings-menu-screen.png
+	:alt: Main settings page for AWX.
```

Comment:
```suggestion
	:alt: The main settings window for AWX.
```

Reply:
Would it be worth keeping this consistent with the preceding wording?

Reply:
Yes thank you!